coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -3210,6 +3218,10 @@ func auditActionFromTransition(transition database.WorkspaceTransition) database
 }
 
 type TemplateVersionImportJob struct {
+	// TemplateID is not guaranteed to be set. Template versions can be created
```

Comment:
> Template versions can be created without being associated with a template.

Can you elaborate on that?
Do we create an initial template version first then associate it with a template?

Reply:
> Do we create an initial template version first then associate it with a template?

Correct, see `cli/templatepush.go` for illustration.

Reply:
Yea, it's been a thorn in my side for a long time that the first template push is an orphaned version :cry: